doc/neps/nep-0035-array-creation-dispatch-with-array-function.rst

```diff
@@ -135,60 +291,43 @@ There are two downsides to the implementation above for C functions:
 2.  To follow current implementation standards, documentation should be attached
     directly to the Python source code.
 
-Alternatively for C functions, the implementation of ``like=`` could be moved
-into the C implementation itself. This is not the primary suggestion here due
-to its inherent complexity which would be difficult too long to describe in its
-entirety here, and too tedious for the reader. However, we leave that as an
-option open for discussion.
-
-Usage
------
+The first version of this proposal suggested the C implementation above as one
+viable solution. However, due to the downsides pointed above we have decided to
+implement that entirely in C. Please refer to [implementation]_ for details.
 
-The purpose of this NEP is to keep things simple. Similarly, we can exemplify
-the usage of ``like=`` in a simple way. Imagine you have an array of ones
-created by a downstream library, such as CuPy. What you need now is a new array
-that can be created using the NumPy API, but that will in fact be created by
-the downstream library, a simple way to achieve that is shown below.
+Alternatives
+------------
 
-.. code:: python
+Recently a new protocol to replace ``__array_function__`` entirely was proposed
+by NEP 37 [6]_, which would require considerable rework by downstream libraries
+that adopt ``__array_function__`` already, because of that we still believe the
+``like=`` argument is beneficial for NumPy and downstream libraries. However,
+that proposal wouldn't necessarily be considered a direct alternative to the
+present NEP, as it would replace NEP 18 entirely, on which this builds upon.
+Discussion on details about this new proposal and why that would require rework
+by downstream libraries is beyond the scopy of the present proposal.
```

Comment:
```suggestion
by downstream libraries is beyond the scope of the present proposal.
```

Is a scopy a little scope? 😂

Reply:
Actually a _tinY_ scopy, inheriting the 'y'. 😄